### PR TITLE
add gflags to switch octal escape char support

### PR DESCRIPTION
**Pull request by czpmango**
Fixes [https://github.com/vesoft-inc/nebula-graph/issues/1033](https://github.com/vesoft-inc/nebula-graph/issues/1033)

### CODE REVIEW

**Review by Aiee — DISMISSED** [review]

File: src/parser/scanner.lex [hunk 1]
```diff
@@ -9,6 +9,9 @@
 #include "parser/GQLParser.h"
 #include "parser/GraphScanner.h"
 #include "GraphParser.hpp"
+#include <gflags/gflags.h>
+
+DEFINE_bool(DisableOctEscapeChar, false, "Octal escape character will be disabled in next version to ensure compatibility with cypher.");
```
Comment (yixinglu) [inline src/parser/scanner.lex:14]: disable_oct_escape_char

Comment (yixinglu) [reply src/parser/scanner.lex:14]: declare in GraphFlags.h

Comment (czpmango) [reply src/parser/scanner.lex:14]: Ok.

**Review by nevermore3 — APPROVED** [review]

**Review by jievince — APPROVED** [review]

**Review by yixinglu — COMMENTED** [review]
Could you add some cases to test this bugfix?

**Review by yixinglu — APPROVED** [review]

**Review by jievince — APPROVED** [review]

**Pull request merged**